db/schema.rb

```diff
@@ -265,7 +265,7 @@
   end
 
   create_table "decidim_awesome_config", force: :cascade do |t|
-    t.jsonb "var"
+    t.string "var"
```

Comment:
Here I can't test if it breaks the awesome config on the current instance

Reply:
@Quentinchampenois 
Where does that change come from ? 
It's strange to have a change in the schema.rb (meaning there was a db:schema:dump) **without** a new migration.

Reply:
I suggest that you reverse this file to keep it outside of the P.R.